frontend/src/app/modules/home/issues/issues.component.ts

```diff
@@ -43,11 +48,18 @@ export class IssuesComponent extends BaseComponent implements OnInit {
         event.originalEvent.stopPropagation();
     }
 
+    private showPaginator() {
+        if (this.length > 10) {
+            this.paginators = (this.paginators === false);
```

Comment:
Perhaps this if statement can be simplified.
```suggestion
        this.paginators = this.length > 10;
```
What do you think about this condition?